packages/api-explorer/src/PathUrl.jsx

```diff
@@ -16,6 +20,57 @@ function splitPath(path) {
       return { type: part.match(/[{}]/) ? 'variable' : 'text', value: part.replace(/[{}]/g, '') };
     });
 }
+
+function renderButtonTry(loading, onSubmit){
+  /* <button
```

Comment:
Small tip: 
If you want refactor later, you can set this as //TODO:. If you want fix, // FIXME. If a comment is to remove old code, delete it. 😄